packages/turbo/src/VisitableView.tsx

```diff
@@ -96,14 +107,34 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
       [viewErrorHandler]
     );
 
+    const handleOnLoad = useCallback(
+      (e: NativeSyntheticEvent<OnLoadEvent>) => {
+        initializeStradaBridge();
+        onLoad?.(e);
+      },
+      [initializeStradaBridge, onLoad]
+    );
+
+    const memoizedStradaComponents = useMemo(
+      () =>
+        stradaComponents?.map(({ StradaReactComponent }, i) => (
+          <StradaReactComponent key={i} useStrada={useStrada} />
+        )),
+      [stradaComponents, useStrada]
```

Comment:
currently `useStrada` returns function on each render, and we should probably wrap it in `useCallback`